composeApp/src/iosMain/kotlin/org/ooni/probe/SetupDependencies.kt

```diff
@@ -64,6 +73,56 @@ import platform.UIKit.popoverPresentationController
 import platform.darwin.NSObject
 import platform.darwin.NSObjectMeta
 
+object PreferenceMigration : DataMigration<Preferences> {
```

Comment:
This is applied not just for testing right? I feel like this will override our `BootstrapPreferences`, because it sets `FIRST_RUN`. Can't we just run this for testing? Through some flag like you have?

Reply:
This is not for testing. To ensure that i can bypass onboarding, I needed to find a way to migrate native preferences to multiplatform which we did not do before. One way forward was to roll a preference migration by hand. Applies only to apps that dont have multiplatform preferences already.

Reply:
Ok, you confirmed the migration doesn't run for new users? Then it's ok.